opencraft/urls.py

```diff
@@ -22,10 +22,14 @@
 
 # Imports #####################################################################
 
+from django.conf import settings
 from django.conf.urls import include, url
 from django.contrib import admin
 from django.views.generic.base import RedirectView
 
+if settings.DEBUG:
+    import debug_toolbar
```

Comment:
This causes a linter error, because of the following import.  I suggest moving this inside the `if` statement at the bottom of the file (as I did when I [temporarily fixed this a while ago](https://github.com/open-craft/opencraft/commit/6fa502f4a60f520fceb008206bba124c9d22d55b)).

Reply:
I put it at the top to avoid a pylint error. When I had it at the bottom of the file, pylint complained:
{code}
************* Module opencraft.urls
C: 48, 4: Import "import debug_toolbar" should be placed at the top of the module (wrong-import-position)
C: 48, 4: external import "import debug_toolbar" comes before "import opencraft.views as views" (wrong-import-order)
{code}
If I make pep8 pass, then pylint complains, and viceversa.
I now put the import at the end as you suggest and see we'll see whether pylint still complains in remote, and repushed.

Reply:
If it doesn't fix, you could temporarily go with `# pylint: disable=wrong-import-position,wrong-import-order`.